code/utils/RandomCCompat.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SCP_RANDOM_C_COMPAT_H
+#define SCP_RANDOM_C_COMPAT_H
+
+// provides a C compatibility layer to the Random class
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
Why the `extern "C"`. All our code is C++ anyway so this seems to be unnecessary.

Reply:
The reason I added the C compatibility layer was to allow us to change the `rand()`/`srand()` calls in the `lib` subfolder (the relevant libraries are written in C) to use the new `Random` class.

But if the team thinks I should leave the files in `lib` alone (that seems to be @notimaginative 's opinion), I'll scrap the C compatibility layer. What do you think?

I don't think libRocket can be integrated, anyway, since we use it via git submodule.

Reply:
I agree with @notimaginative, stuff in lib should not be changed so this can be removed.